tensorboard/compat/proto/BUILD

```diff
@@ -25,9 +25,12 @@ tb_proto_library(
         "event.proto",
         "function.proto",
         "graph.proto",
+        "layout.proto",
```

Comment:
I don't think these protos are required  for the writer implementation. Can we just remove those additions from this PR for now?

Reply:
layout.proto is needed by `custom_scalar` plugin

Reply:
@lanpa Yes, but why is that relevant to the writer implementation?

Reply:
@nfelt @orionr  Oh, I think I got your point. Do you mean these protos are not relevant to the writer and should be added in another PR?

Reply:
Exactly right - thanks.